monitor-iot-data-with-freeboard/README.md

```diff
@@ -0,0 +1,210 @@
+}
+```
+
+Launching `sin-generator-device.sh` will start creating documents in Kuzzle. You should be able to see the documents using *Kuzzle Admin Console*
+
+![alt text](img/kuzzle-sin-gen-docs.png "Sinus generator device documents")
+
+To monitor this device's *value* field, in *freeboard*, we will create a datasource that will monitor documents in `fb-howto-index/fb-howto-collection` and subscribe to document for which the `device_id` field equals *`SIN_GENERATOR`*
+
+In *freeboard*, under **DATASOURCES**, click the **ADD** button, choose Kuzzle Datasource. Enter settings as explained in previous section.
+
+Now, for the filter, as we want to match only created documents whose `device_id` is *`SIN_GENERATOR`*, we will use the following filer:
+
+```json
+{
+    "equals" : {
+        "device_id": "SIN_GENERATOR"
+    }
+}
+```
+
+Give a name to this datasource: *MySinGenerator*
+
+Click **SAVE**. *MySinGenerator* should now appear in your **DATASOURCES** and be updated regularly.
+
+### Step 4: Adding a widget to visualize data from SIN_GENERATOR device
+
+We will use the *Sparkline* widget to visualize the SIN_GENERATOR device `value` field.
+
+- First add a new pane by clicking the **ADD PANE** button
+- In the newly created pane, click the **+** button to add a new widget
+- Select *Sparkline* widget
+- Give a title to the widget: *My Sin Generator*
+- In the `VALUE` field, enter *`datasources["MySinGenerator"]["value"]`*
+
+![alt text](img/sin-gen-sparkline.png "Sparkline settings")
+
+Click **SAVE**
+
+You should start visualizing the evolution of the `SIN_GENERATOR` device over time:
+
+![alt text](img/sin-gen-sparkline-viz.png "SinGenerator device spakline")
+
+### Step 5: Add other devices
+
+Next step is to add data from another virtual data. You can use the `./square-generator-device.sh` that will generate a square function for a device with `device_id` set to *`SQUARE_GENERATOR`*
+
+You can follow  steps 3 and 4 using *`SQUARE_GENERATOR`* in place of *`SIN_GENERATOR`*
+
+Both scripts accept a argument that will be used as `device_id` field in the created document, allowing to simulate several differents generator devices.
+
+```console
+$ ./square-generator-device.sh my_square_gen_id
+Generating square function as "device_id" : my_square_gen_id
+```
+
+![alt test](img/dashboard.png "Dashboard")
+
+## Conclusion
+
+Using [Kuzzle](http://www.kuzzle.io) Realtime capabilities, [Kuzzle Datasource freeboard plugin](http://www.github.com/kuzzleio/kuzzle-freeboard-plugin), and the correct [subscription](https://docs.kuzzle.io/sdk-reference/collection/subscribe/) filter, you are able to monitor/vizualize any realtime parametter of your Kuzzle based IoT application.
+
+For now, Kuzzle Freeboard Datasource plugin is limited to monitoring document when they are **created**, but you could easily adapt it to monitor document when they are **updated**, **deleted**, or **published** for pure realtime documents (documents that are not stored).
```

Comment:
...monitoring documents... (used twice)....real-time documents.

what do yo mean by pure ? I would probably just remove that, i.e **published** for real-time documents.